Tests/test_MMCIF.py

```diff
@@ -203,6 +203,52 @@ def test_filehandle(self):
         structure = parser.get_structure("example", open("PDB/1A8O.cif"))
         self.assertEqual(len(structure), 1)
 
+    def test_point_mutations(self):
+        """Test if the parser can parse point mutations correctly."""
+
+        def _run_parser(parser):
+            structure = parser.get_structure("example", "PDB/3JQH.cif")
+
+            # Residue 1 and 15 should be disordered.
+            res_1 = structure[0]["A"][1]
+            res_15 = structure[0]["A"][15]
+
+            # Cursory check -- this would be true even if the residue just
+            # contained some disordered atoms.
+            self.assertTrue(res_1.is_disordered(), "Residue 1 is disordered")
+            self.assertTrue(res_15.is_disordered(), "Residue 15 is disordered")
+
+            # Check a non-mutated residue just to be sure we didn't break the
+            # parser and cause everyhing to be disordered.
+            self.assertFalse(
+                structure[0]["A"][13].is_disordered(),
+                "Residue 13 is not disordered")
+
+            # Check that the residue types were parsed correctly.
+            self.assertSetEqual(
+                set(res_1.disordered_get_id_list()),
+                {"PRO", "SER"},
+                "Residue 1 is proline/serine")
+            self.assertSetEqual(
+                set(res_15.disordered_get_id_list()),
+                {"ARG", "GLN", "GLU"},
+                "Residue 15 is arginine/glutamine/glutamic acid")
+
+            # Quickly check that we can switch between residues and that the
+            # correct set of residues was parsed.
+            res_1.disordered_select('PRO')
+            self.assertAlmostEqual(
+                res_1["CA"].get_occupancy(),
+                0.83, 2, "Residue 1 proline occupancy correcy")
+
+            res_1.disordered_select('SER')
+            self.assertAlmostEqual(
+                res_1["CA"].get_occupancy(),
+                0.17, 2, "Residue 1 serine occupancy correcy")
+
+        _run_parser(MMCIFParser(QUIET=True))
+        _run_parser(FastMMCIFParser(QUIET=True))
+
```

Comment:
Personally I'd restructure this test to have two test methods (normal and fast), something like this (untested):

``` python
    def _run_parser(self, parser):
        """Internal function for common test code."""
        structure = parser.get_structure("example", "PDB/3JQH.cif")
        # ...

    def test_point_mutations_main_parser(self):
        """Point mutations using MMCIFParser"""
        self._run_parser(MMCIFParser(QUIET=True))

    def test_point_mutations_fast_parser(self):
        """Point mutations using FastMMCIFParser"""
        self._run_parser(FastMMCIFParser(QUIET=True))
```

This means ``MMCIFParser`` and ``FastMMCIFParser`` will get tested separately (useful if for example ``MMCIFParser`` was breaking meaning ``FastMMCIFParser`` would not be tested). i.e. General principle of making individual tests smaller

But as it is looks fine.